miso-web/src/main/webapp/scripts/urls.js

```diff
@@ -957,12 +957,12 @@ Urls = (function () {
   };
 
   // Service Records
-  var serviceRecordUiBase = baseUrl + "/instrument/servicerecord";
+  var serviceRecordUiBase = baseUrl + "/instrument/" + instrumentId + "/servicerecord";
   ui.serviceRecords = {
     edit: idUrlFunction(serviceRecordUiBase),
   };
 
-  var serviceRecordRestBase = restBase + "/servicerecords";
+  var serviceRecordRestBase = restBase + "/instrument/" + instrumentId + "/servicerecords";
```

Comment:
`instrumentId` is undefined, and cannot be defined here. The values and functions within `ui.serviceRecords` and `rest.serviceRecords` will have to be modified to take an `instrumentId` parameter and construct the full URL